domains/certificates/Query.API/API.UnitTests/GranularCertificateIssuer/EnergyMeasuredConsumerTest.cs

```diff
@@ -70,7 +70,7 @@ public async Task Consume_ConsumptionPoint_NoEventsSaved()
     [Fact]
     public async Task Consume_MeteringPointNotOnboarded_NoEventsSaved()
```

Comment:
```suggestion
    public async Task Consume_OneDayBeforeIssuranceStartDate_NoEventsSaved()
```

Reply:
Is your suggestion to inject MockMasterDataCollection into SyncState?